packages/contracts/contracts/THUSDToken.sol

```diff
@@ -240,6 +253,12 @@ contract THUSDToken is Ownable, CheckContract, ITHUSDToken {
         _burn(_account, _amount);
     }
 
+    // --- thUSD property getters ---
+
+    function getGovernanceTimeDelay() external view override returns (uint256) {
+        return governanceTimeDelay;
+    }
+
```

Comment:
`governanceTimeDelay` is already `public` which means there is getter for that field

Reply:
Addressed in [refactor: add governance time delay in thusd constructor](https://github.com/Threshold-USD/dev/pull/113/commits/9f771a49a984e3122ea25dd5881305289819e1f9)